src/github_type.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Type of returned data from GitHub web API."""
```

Comment:
s/Type of returned data from/Return types for the

Reply:
Minor comments